src/app/onprem/license-nag-message-management/nag-message.service.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2015-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ */
+'use strict';
+
+
+/**
+* This class is handling the service for the nag message
+* @author Oleksii Orel
+*/
+export class NagMessageService {
+
+  /**
+   * Default constructor that is using resource injection
+   * @ngInject for Dependency injection
+   */
+  constructor($document, cheBodyInjectorSvc , codenvyLicense) {
+    'ngInject';
+
+    this.$document = $document;
+    this.cheBodyInjectorSvc = cheBodyInjectorSvc;
+    this.codenvyLicense = codenvyLicense;
+
+  }
+
+  /**
+   * Show nag message
+   */
+  showNagMessage() {
+    this.$document.find('body').addClass('nag-message-indent');
```

Comment:
We use body element in our high level service(nagMessageService) to add a new class with style "top: 30px". It is because cheBodyInjectorSvc is universal service for adding any new element with id. In this case, when we add a nag message an the top of the screen, we should add a new style for <body> to move it down. We need it to make free space for nag message with style "position: fixed"(we always show nag message in the top of the screen).

Do you have another solution for this problem?